packages/atomic/src/components/search/facets/facet-value-box/facet-value-box.tsx

```diff
@@ -21,7 +21,7 @@ export const FacetValueBox: FunctionalComponent<FacetValueProps> = (
     <li key={props.displayValue}>
       <Button
         style="outline-bg-neutral"
-        part="value-box"
+        part={`value-box${props.isSelected ? ' selected-value' : ''}`}
```

Comment:
Since multiple values can be the "selected value" (as far as I'm aware at-least), I would have both a selected and unselected part. I would also prefer to have both parts be prefixed with `value-box` to reflect that they are on the same element (e.g., `value-box-unselected`).